install/deploy_final_non-compute_node.md

```diff
@@ -434,7 +435,17 @@ it is used for Cray installation and bootstrap.
     /tmp/csi handoff bss-update-param --set metal.no-wipe=1
     ```
 
-## 6. Configure DNS and NTP on each BMC
+## 6. Update select RPMs
+
+(`ncn-m001`) Run the following command to ensure that select RPMs on `ncn-m001` are at the correct version.
+
+```bash
+/usr/share/doc/csm/upgrade/scripts/upgrade/util/upgrade-test-rpms.sh --local
+```
+
+If the output ends with `PASSED`, then it was successful, despite any warning messages that may have been displayed.
```

Comment:
m001 will grab the latest RPMs based on cloud-init, all that's missing in cloud-init right now is `craycli`. I think all you have to do is add craycli to https://github.com/Cray-HPE/csm/blob/release/1.6/rpm/cloud-init.yaml

Reply:
https://github.com/Cray-HPE/csm/pull/2973